src/partials/footer.hbs

```diff
@@ -11,7 +11,7 @@
     </a>
     <p class="!m-0">
       <span>©</span>
-      <span id="copyrightdate">2024</span>
+      <span id="copyrightdate">2025</span>
```

Comment:
@eric-schneider Should we use the handlebars helper `year` here? It's already set up in the helpers folder from the default UI, or was there a reason we switched to manually setting it? To use the current year this line can be changed to
```
<span id="copyrightdate">{{year}}</span>
```

Reply:
> was there a reason we switched to manually setting it?

I don't think we actually "switched" to manually setting the year, because even though that helper ships with the [Antora Default UI](https://gitlab.com/antora/antora-ui-default), I don't see any of the default `.hbs` files actually referencing it. It looks like we've been manually updating the year ever since it was originally added in [PR 11](https://github.com/riptano/docs-ui/commit/6aa499f53887d411eacee3481d34395b486f1bf3#diff-94f959b9f08a0acf70bbc09fd370de625269834e3b5a6ad922c452a1562c87e9).

Regardless, **_YES_**, we should definitely be using the `year` helper! 😆 Thanks for calling this out!! I'll push this change in my next commit.